app/assets/javascripts/tylium/modules/resource_sidebar.js

```diff
@@ -0,0 +1,24 @@
+document.addEventListener('turbolinks:load', function() {
+
+  if ($('[data-behavior~=resource-sidebar]').length) {
+    const $resourceSidebar = $('[data-behavior~=resource-sidebar]'),
+          $resourceSidebarToggle = $('[data-behavior~=resource-sidebar-toggle]'),
+          localStorageKey = 'project.ce.resource-sidebar-collapsed',
+          savedState = localStorage.getItem(localStorageKey);
+
+    function toggleResourceSidebar() {
+      $resourceSidebar.toggleClass('collapsed');
+      $resourceSidebarToggle.find('i').toggleClass('fa-chevron-right fa-chevron-left');
+      localStorage.setItem(localStorageKey, $resourceSidebar.hasClass('collapsed'));
+      Turbolinks.clearCache();
```

Comment:
What's the reason for clearing the cache here? It seems weird to lose the turbolinks speed when toggling the resource sidebar.

Reply:
This is the same approach as the main sidebar, if we don't clear the cache there is a weird jump in the view where the sidebar reverts to the default state for a second and then back to what the user had it set to.